### PR TITLE
feat: Custom backendUrl in invite-member.usecase and resend-invite.usecase #3934

### PR DESCRIPTION
**Pull request by pritipsingh**
### What change does this PR introduce?

More flexibility for injecting a custom backendUrl for invite-member.usecase and resend-invite.usecase
I have added config in all the interfaces for invites, resend & bulk and passed it in. Password one is left but just wanted to get it reviewed first.

### Why was this change needed?

Closes [3934](https://github.com/novuhq/novu/issues/3934)

### CODE REVIEW

File: apps/api/src/app/auth/usecases/password-reset-request/password-reset-request.usecase.ts [hunk 1]
```diff
@@ -14,16 +14,22 @@ export class PasswordResetRequest {
...
+    private userRepository: UserRepository,
+    private memberRepository: MemberRepository
+  ) {}
 
   async execute(command: PasswordResetRequestCommand): Promise<{ success: boolean }> {
     const email = normalizeEmail(command.email);
     const foundUser = await this.userRepository.findByEmail(email);
     if (foundUser && foundUser.email) {
       const { error, isBlocked } = this.isRequestBlocked(foundUser);
+      //  const member = await this.memberRepository.findById(foundUser._id);
```
Comment (markmssd) [inline apps/api/src/app/auth/usecases/password-reset-request/password-reset-request.usecase.ts:28]: Curious if keeping this commented is intended 😸

Comment (pritipsingh) [reply apps/api/src/app/auth/usecases/password-reset-request/password-reset-request.usecase.ts:28]: yup haven't implemented it for passwords. cannot find the member just by id. and there is no fn for `findById` in `memberRepository`  we could use others but will need more for it

Comment (scopsy) [reply apps/api/src/app/auth/usecases/password-reset-request/password-reset-request.usecase.ts:28]: Could you please remove this code if it's not in use in this file? Including the imports part

**Comment by scopsy** [comment]
@pritipsingh i'm not sure regarding the approach of passing this information from the client. Why not just having an env variable the backend can have to connect with the proper backend URL by it?

**Comment by mirzap** [comment]
Any news on this?

**Comment by scopsy** [comment]
Closing due to inactivity

**Pull request closed without merging**